src/Commands/WorkflowDumpCommand.php

```diff
@@ -69,6 +71,10 @@ public function handle()
                 ' Please specify a valid support class with the --class option.');
         }
 
+        $dumper_options = [
```

Comment:
Let's match the style in this codebase of camel-cased names
```suggestion
        $dumperOptions = [
```